### PR TITLE
Add loglevel for msg output

### PR DESCRIPTION
**Pull request by florianb**
Sorry for the delay.

I added the `FLATPAK_IDE_LOGLEVEL` variable and tested the functionality locally. I would be happy if you could take a look and let me know if there's something i should change or add.

Thank you very much.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 
 Current functions:
+
 * Show up a readme on first launch
 * Show up a notice after SDK update
 * Enable installed SDK extensions (from `/usr/lib/sdk`)
   * `FLATPAK_ENABLE_SDK_EXT` must be set to a comma-separated list or `*`
 * Enable installed development tools (from `/app/tools`)
 * Isolate npm/pip/cargo/etc packages from host environment
+* Supress log messages setting `FLATPAK_IDE_LOGLEVEL` to `0` (defaults to `1`)
```
Comment (gasinvein) [inline README.md:13]: I feel it doesn't belong to this list, which enumerated supported features. Maybe add another "Configuration options" list, and put this variable there?

Comment (florianb) [reply README.md:13]: Good point – expect a change within the next 24 hours.. :grimacing: :runner:

Comment (florianb) [reply README.md]: I added a env-var section. I always prefer a explicit documentation but you know markdown is a bit limited in that regard. However, it should be easy maintainable and can be populated as needed in the future. I added the currently used env vars (and their defaults) to not unintentionally hide them.

File: README.md [hunk 2]
```diff
@@ -132,3 +133,29 @@ To this:
...
+
+### FLATPAK_PREFER_USER_NPM (= `FLATPAK_PREFER_USER_PACKAGES`)
+
+### FLATPAK_ISOLATE_CARGO (= `FLATPAK_ISOLATE_PACKAGES`)
+
+### FLATPAK_PREFER_USER_CARGO (= `FLATPAK_PREFER_USER_PACKAGES`)
+
+### FLATPAK_ISOLATE_PIP (= `FLATPAK_ISOLATE_PACKAGES`)
+
+### FLATPAK_PREFER_USER_PIP (= `FLATPAK_PREFER_USER_PACKAGES`)
```
Comment (gasinvein) [inline README.md:161]: Environment variable names look pretty bad as section headers. Maybe just change it to a list?

```suggestion
* `FLATPAK_IDE_LOG_LEVEL` (= `1`)

  Controls verbosity of the module, `0` supresses all module-originated outputs to stdout.

  * Type: Number (`0` | `1`)

* `FLATPAK_PREFER_USER_PACKAGES` (= `0`)

* `FLATPAK_ISOLATE_PACKAGES` (= `1`)

* `FLATPAK_ISOLATE_NPM` (= `FLATPAK_ISOLATE_PACKAGES`)

* `FLATPAK_PREFER_USER_NPM` (= `FLATPAK_PREFER_USER_PACKAGES`)

* `FLATPAK_ISOLATE_CARGO` (= `FLATPAK_ISOLATE_PACKAGES`)

* `FLATPAK_PREFER_USER_CARGO` (= `FLATPAK_PREFER_USER_PACKAGES`)

* `FLATPAK_ISOLATE_PIP` (= `FLATPAK_ISOLATE_PACKAGES`)

* `FLATPAK_PREFER_USER_PIP` (= `FLATPAK_PREFER_USER_PACKAGES`)
```

Comment (florianb) [reply README.md:161]: Yeah - sure :smile:
I have to admit i had a list before.

**Comment by gasinvein** [comment]
Looks good, thanks.

**Pull request merged**